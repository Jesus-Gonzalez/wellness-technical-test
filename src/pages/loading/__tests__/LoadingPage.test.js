import React from 'react'
import renderer from 'react-test-renderer'

import LoadingPage from '../LoadingPage'

describe('pages', () => {
  describe('LoadingPage', () => {
    it('should render', () => {
      const Component = renderer.create(<LoadingPage />)

      expect(Component).toMatchSnapshot()
    })
  })
})
