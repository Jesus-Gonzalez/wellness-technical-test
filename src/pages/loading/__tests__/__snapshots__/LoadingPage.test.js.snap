// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages LoadingPage should render 1`] = `
<div
  className="d-flex justify-content-center w-100"
>
  <div
    className="spinner"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
